freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_install.sh

```diff
@@ -26,8 +21,8 @@ ipa-server-install \
           --allow-zone-overlap \
           --ssh-trust-dns \
           --mkhomedir \
-          --ip-address "$IPADDR" \
-          $FORWARDERS \
```

Comment:
Adding @jamisonbennett to the loop.

d45bd413648e564b09a33bde221fde7d06c6479b commit changed the auto forwarders to link-local IP address as a forwarder which fails with the following in GCP ->
"ipa-server-install: error: option --forwarder: invalid IP address 169.254.169.254: cannot use link-local IP address 169.254.169.254"

More info https://unix.stackexchange.com/questions/222346/ipa-server-configuration-cannot-use-ip-network-address

Reply:
I was relying on one line from the commit message "Use the same logic for the master too eventhough it didn't matter.
"